libs/atqr/domain/src/lib/domain-spec/stories/playerReceivesCondolences.disl

```diff
@@ -0,0 +1,31 @@
+read_also api.disl
+read_also enums.disl
+read_also ui.disl
+
+ChallengeFailed!
+    rules:
+        - JudgementStatus = Judged
```

Comment:
= means assignment. Either use == or `is`. Also you are using the enum name as if it were the property.
The correct way to express what you meant is `challenge's judmentStatus is Judged`

Reply:
Challenge.@judges JudgementStatus == Judged 
is this right ?